app/services/google/keyword_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Google
+  class KeywordService
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                 'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36'
+
+    BASE_SEARCH_URL = 'https://www.google.com/search'
+
+    def initialize(keyword, lang: 'en')
+      @escaped_keyword = CGI.escape(keyword)
+      @url = "#{BASE_SEARCH_URL}?q=#{@escaped_keyword}&hl=#{lang}&gl=#{lang}"
```

Comment:
(The bellow is a totally optional and opinionated suggestion)

I generally prefer performing the least in the initialize method. This allows to delay some calls (e.g. formatting the URL) and to only call them when we are sure we will consume their output.

This would look like:

```
    def initialze(keyword, lang: :en)
      @keyword = keyword
      @lang = lang
    end

    def call
      html = WebpageFetchService.new(url, USER_AGENT).call

      ParserService.new(html.body).call if html
    end

    private

    def url
      escaped_keyword = CGI.escape(keyword)

      "#{BASE_SEARCH_URL}?q=#{@escaped_keyword}&hl=#{lang}&gl=#{lang}"
    end
```

In a simple service like this one, it does not change much. But when the class becomes more complex, this can make a big difference ;)